test/e2e/kubectl/kubectl.go

```diff
@@ -1990,7 +1990,22 @@ metadata:
 			outBuiltIn := e2ekubectl.RunKubectlOrDie("", "get", "nodes", node.Name)
 			ginkgo.By(fmt.Sprintf("calling kubectl get nodes %s --subresource=status", node.Name))
 			outStatusSubresource := e2ekubectl.RunKubectlOrDie("", "get", "nodes", node.Name, "--subresource=status")
-			gomega.Expect(outBuiltIn).To(gomega.Equal(outStatusSubresource))
+			// Avoid comparing values of fields that might end up
+			// changing between the two invocations of kubectl.
+			requiredOutput := [][]string{
+				{"NAME"},
+				{"STATUS"},
+				{"ROLES"},
+				{"AGE"},
+				{"VERSION"},
+				{node.Name},                           // check for NAME
+				{""},                                  // avoid comparing STATUS
+				{""},                                  // avoid comparing ROLES
+				{""},                                  // avoid comparing AGE
```

Comment:
these don't really make sense with what checkOutput does...

Reply:
Yeah I agree it's a little hacky to workaround. I'll send a follow up